driver-core/src/main/java/com/datastax/driver/core/Configuration.java

```diff
@@ -80,6 +85,7 @@ public Configuration(Policies policies,
         this.metricsOptions = metricsOptions;
         this.queryOptions = queryOptions;
         this.nettyOptions = nettyOptions;
+        this.codecRegistry = codecRegistry;
     }
```

Comment:
Let's introduce a builder for this class, like for `Policies`. This will avoid backward-compatibility issues with the constructors.